cartography/intel/aws/ec2/instances.py

```diff
@@ -255,8 +302,8 @@ def cleanup_ec2_instances(neo4j_session: neo4j.Session, common_job_parameters: D
 
 @timeit
 def sync_ec2_instances(
-    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
```

Comment:
nothing changed here, leave as is plz

Reply:
PyCharm keeps indenting it back as it crosses the 120 characters per line limit.

Reply:
change it in the config?

Reply:
It's also a linter requirement. I'll fail the linter test if the line exceeds 120 characters per line.

Reply:
oh ok coool